src/components/MerchantPayout/VoucherStatusModal/VoucherStatusModal.tsx

```diff
@@ -88,27 +89,35 @@ export const VoucherStatusModal: FunctionComponent<VoucherStatusModal> = ({
   const isVisible = checkValidityState === "CHECKING_VALIDITY";
 
   let card;
+
+  const title = i18n.t(`errorMessages.notEligible.title`);
+  const details = i18n.t(`errorMessages.notEligible.body`);
+
   if (error instanceof ScannerError) {
     requestAnimationFrame(() => {
       requestAnimationFrame(() => {
-        showAlert("Error scanning", error.message, "Continue scanning", onExit);
+        showAlert(
+          i18n.t(`errorMessages.errorScanning.title`),
+          error.message,
+          i18n.t(`errorMessages.errorScanning.primaryActionText`),
+          onExit
+        );
       });
     });
     return null;
   } else if (error instanceof LimitReachedError) {
     requestAnimationFrame(() => {
       requestAnimationFrame(() => {
-        showAlert("Scan limit reached", error.message, "OK", onExit);
+        showAlert(
+          i18n.t(`errorMessages.scanLimitReached.title`),
+          error.message,
+          i18n.t(`errorMessages.errorScanning.primaryActionText`),
+          onExit
+        );
```

Comment:
Did we miss out refactoring this?

Reply:
This has been extracted out to be shown on the `MerchantPayoutScreen` and now uses the new `showErrorAlert`.